ui/src/components/CopyToClipboard/CopyToClipboard.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { Button } from 'react-bootstrap';
+import { CopyToClipboard as TriggerCopyToClipboard } from 'react-copy-to-clipboard';
+import { MdContentCopy } from 'react-icons/md';
+
+import TooltipTrigger from '../TooltipTrigger';
+import styles from './CopyToClipboard.module.css';
+
+export default function CopyToClipboard(props) {
+  const { id, tittle, text, iconColor = 'black' } = props;
+
+  const [copied, setCopied] = useState(false);
+
+  function onCopy() {
+    setCopied(true);
+    setTimeout(() => setCopied(false), 1100);
+  }
+
+  return (
+    <TooltipTrigger
+      placement={copied ? 'auto' : 'right'}
+      rootClose={false}
+      id={id}
+      tooltipContent={
+        copied ? `${tittle} Copied` : `Copy ${tittle} to Clipboard`
```

Comment:
I wonder if it's really necessary to show this `title` in the tooltip. Would "Copy to clipboard" (lowercase "clipboard" if possible) and "Copied" suffice?

Reply:
yeah not necessary, but a bit more informative, just  Copied feel more advance user. 
we have this component on another application and some users prefer details.